tests/converter/test_positioning.py

```diff
@@ -1,30 +1,30 @@
+import pytest
 from converter import positioning
 from converter.errors import Fig2SketchWarning
 from math import nan
-import pytest
-
 
-def test_nan():
-    fig = {
-        "transform": positioning.Matrix([[nan, nan, nan], [nan, nan, nan]]),
-        "size": {"x": 1, "y": 2},
-    }
-    with pytest.raises(Fig2SketchWarning) as e:
-        positioning.convert(fig)
 
-    assert e.value.code == "POS001"
+class TestConvert:
```

Comment:
It seems we follow the guideline of having test class for public methods and then test functions for each case, so I brought a couple of test files to this _standard_.

Reply:
I just mixed them up, and use a class when grouping more than one test. I usually start as standalone function and create a class once there are some related tests to group. I don't mind that much anyway, so this looks good (and these two tests are related anyway)